src/SFA.DAS.PR.Application/Mediatr/Behaviours/ValidationBehaviour.cs

```diff
@@ -36,13 +36,17 @@ public async Task<TResponse> Handle(TRequest request, RequestHandlerDelegate<TRe
 
                 var responseType = typeof(TResponse);
 
-                if (responseType.IsGenericType)
+                if (responseType.IsGenericType || responseType == typeof(ValidatedBooleanResult))
```

Comment:
My preference here is to improve readability and reduce logical complexity and leave a the 'if (responseType.IsGenericType)' as is, and add a second if statement below that starts 'If (responseType== typeof(ValidatedBooleanResult)...' and do the new code in that.  However, there may be some reason they needed to be combined.  Can you review and try it, and post comments if there are reasons it doesn't work? ta.